### PR TITLE
Add usage tips

### LINKED ISSUE #19: paths.src.projectSass not used during complile
When running compile noticed the files in paths.src.projectSass are not called.

Some more background.  We are hoping to have init output all of the uswds files (img, fonts, scss, and js) files into an assets folder.

Then in our theme folder we can import the scss file at will without editing it.  Which makes updating much easier later.  But currently uswds.paths.dist.theme is basically serving 2 purposes and think should be broken up into 

uswds.paths.dist.theme
 and new path uswds.paths.dist.sass

### PR DESCRIPTION
**Pull request by thisisdano**
Adds a `Usage tips` section to help outline how to best use USWDS Compile in a project. In general, these explain how best to set up a project so you can include all the project files you needs, compile to a single CSS file, and update USWDS without clobbering any project files.

- Use `init` only once.
- Update USWDS assets with `copyAssets`.
- Compile only from a single Sass entry point.
- Only check theme files and custom icons into version control.

This also moves `Usage tips` and `Running the compile functions` up into the `Usage` section.

Fixes #19

### CODE REVIEW

File: README.md
Comment (amyleadem) [inline README.md:139]:
```suggestion
- **Compile only from a single Sass entry point.** Define the location of this entry point with `paths.dist.theme`. If you have project Sass files outside the `paths.dist.theme` directory, load these files into this single entry point via `@forward`, `@use`, or `@import`. Set `paths.src.projectSass` to your project Sass parent directory if you want to include these files in your `gulp watch` task. The Sass will still compile from the single entry point located in `paths.dist.theme`.
```

File: README.md
Comment (amyleadem) [inline README.md:140]:
```suggestion
- **Only check theme files and custom icons into version control.** You should have a build process that copies static assets like images, fonts, and compiled JavaScript from the `uswds` package. This assures that these assets are up-to-date with whatever version of USWDS you're using. You only need to track your customizations (like settings, theme files, custom icons, and your gulpfile) in version control.  
```

Looks like `gulpfile` is typically lowercase: https://gulpjs.com/docs/en/getting-started/javascript-and-gulpfiles/

File: README.md
Comment (amyleadem) [inline README.md:139]:
```suggestion
- **Compile only from a single Sass entry point.** Define the location of this entry point with `paths.dist.theme`. If you have project Sass files outside the `paths.dist.theme` directory, load these files into your single entry point via `@forward`, `@use`, or `@import`. To include these project Sass files in your `gulp watch` task, set `paths.src.projectSass` to your project Sass directory. The Sass will still compile from the single entry point located in `paths.dist.theme`.
```

This section makes sense as is, but wanted to offer up a few suggestions that might help clarify meaning.

**Review by amyleadem — APPROVED** [review]
Helpful updates! I added a couple of suggestions in the comments, but none are necessary for moving forward.

File: README.md
Comment (amyleadem) [inline README.md:137]:
```suggestion
- **Use `init` only once.** The `init` task is meant for initializing the design system on a project. Since it will overwrite project files (like settings files and the Sass entry point), use it sparingly and don't use it for updating the design system on a project, or at any point after you've customized your settings files.  
```

Whoops, found a small typo after I approved! This one _should_ be fixed before merging :)

**Review by mejiaj — COMMENTED** [review]
LGTM after @amyleadem's suggestions.

**Review by mejiaj — APPROVED** [review]

**Pull request merged**
